core/src/impl/Kokkos_Half_MathematicalFunctions.hpp

```diff
@@ -315,6 +320,88 @@ KOKKOS_INLINE_FUNCTION bool isnan(Kokkos::Experimental::bhalf_t x) {
   return Impl::impl_isnan(x);
 }
 #endif
+
+#if !(defined(KOKKOS_ENABLE_CUDA) && defined(KOKKOS_COMPILER_MSVC))
+template <typename fp16_t>
+KOKKOS_INLINE_FUNCTION fp16_t nextafter_half_impl(fp16_t from, fp16_t to) {
```

Comment:
You do not raise any FE exceptions. Is it ok to do so, and in this case, can you add a comment?

Reply:
We can abort these cases?
```
1. from is max, to is pos_inf -> return pos_inf (FE_INEXACT and FE_OVERFLOW)
2. from is min, to is neg_inf -> return neg_inf (FE_INEXACT and FE_OVERFLOW)
3. from is pos, to is neg -> return pos_dn or +0 (FE_INEXACT and FE_UNDERFLOW)
4. from is neg, to is pos -> return neg_dn or -0 (FE_INEXACT and FE_UNDERFLOW)
```

Reply:
This does not raise FE exception
https://godbolt.org/z/1bnEsxMKf

Reply:
I confirmed

https://godbolt.org/z/1jbeW7YTK

Reply:
I would not bother

Reply:
Can you just add a comment stating that you do not raise any FE?